docs/Course Wiki/ECE Course Offerings/ECE342.md

```diff
@@ -0,0 +1,54 @@
+
+## Course Structure
+
+ECE342 has an average workload.  Homework is assigned weekly, except during exam weeks.  The length of each homework set varies over the course of the class.  Towards the beginning and end of the course, homeworks are challenging but manageable and students can expect to spend a few hours on them.  However, during the middle of the course, homeworks can be quite challenging and time consuming if you do not have a full understanding of the underlying concepts.  For this reason, it is recommended to start early on the homework and attend office hours.  
+
+In addition to the homeworks, the course has three midterm exams and one final exam.  The difficulty of these exams varies depending on the instructor, but expect them to be challenging.  Reviewing the homeworks and provided past exams is important to doing well on the midterms and final.    
+
+## Instructors
+
+This course is taught by a couple of different professors who are well-known in the field of electronic circuits.  In recent semesters, Professors Hanumolu, Radhakrishnan, Rosenbaum, Shanbhag, Zhou, Kim, and Schutt-Aine have taught the class.
+
+## Course Tips
+
+Some homeworks can be decently time consuming and difficult to approach, so starting early and going to office hours is a good idea.  Try not to fall behind, as many concepts in this course build upon previous ones.  Exam problems are often similar to homework problems and past exam problems, so it is recommended to work through your previous homeworks and the past exams deliberately when studying for exams.  
+
+## Life After
+
+Students who have taken and enjoyed ECE342 are open to a wide range of choices from the different subdisciplines of ECE.  For students interested in integrated circuits, [ECE483 - Analog IC Design](ECE483.md) and [ECE482 - Digital IC Design](ECE482.md) are good follow-up courses.  This course is also an official prerequisite for [ECE453 - Wireless Communication Systems](ECE453.md), for students interested in radio frequency engineering, and [ECE464 - Power Electronics](ECE464.md), for students interested in power and energy systems.  Additionally, students who enjoyed this course should consider [ECE444 - IC Device Theory & Fabrication](ECE444.md) (prerequisite: [ECE340](ECE340.md)) and [ECE425 - Intro to VLSI System Design](ECE425.md) (prerequisite: [ECE385](ECE385.md) or [CS233](../CS%20Course%20Offerings/CS233.md)).
```

Comment:
Switch the order of 482 and 483 so its in numerical order. Switch the order of 425 and 444 so its in numerical order. Also, would these courses only be enjoyable to students who enjoyed the latter part of the course (CMOS stuff), or would these courses be enjoyable to students who enjoyed any of the content in the course?
Also delete CS233? I believe it's equivalent to ECE120, and therefore not worth taking for ECE students.